gms_app/tests.py

```diff
@@ -12,51 +12,65 @@
 def _prepare_db(django_db_setup, django_db_blocker):
     # Set up non-modified objects used by all test methods
     with django_db_blocker.unblock():
-        garden = models.Garden.objects.create(name="Test Garden", description="Test Garden Description")
-        garden_bed = models.GardenBed.objects.create(
-            garden=garden, name="Test Garden Bed", description="Test Garden Bed Description 1"
+        garden = Garden.objects.create(
+            name="Test Garden",
+            description="Test Garden Description",
+        )
+        garden_bed = GardenBed.objects.create(
+            garden=garden,
+            name="Test Garden Bed",
+            description="Test Garden Bed Description 1",
         )
         log.info("Garden '%s' - garden bed '%s'", garden.name, garden_bed.name)
         for plant_i in range(3):
-            p = models.Plant(name=f"Test Plant {plant_i}", description=f"Test Plant Description{plant_i}")
-            p.save()
-            p.garden_bed.add(garden_bed)
-            log.info("\tAdded plant '%s'", p.name)
+            plant = Plant.objects.create(
+                name=f"Test Plant {plant_i}",
+                description=f"Test Plant Description{plant_i}",
+            )
+            planting = Planting.objects.create(
+                plant=plant,
+                garden_bed=garden_bed,
+                location={"x": plant_i, "y": 0},
```

Comment:
@LeskoIam Not sure what you were looking for in the location so took a guess at "x" and "y". Also "row" "col" seem just as if not more appropriate, but we can change that easily enough

Reply:
It is meant to represent the plants location on the garden bed. I'm not sure how location will be stored as I don't yet know what frontend will be used to display garden layout.